ignite/pkg/cosmosclient/cosmosclient.go

```diff
@@ -241,14 +243,23 @@ func (r Response) Decode(message proto.Message) error {
 		return err
 	}
 
-	resData := txMsgData.Data[0]
+	// check deprecated Data
+	if len(txMsgData.Data) != 0 {
+		resData := txMsgData.Data[0]
+		return prototypes.UnmarshalAny(&prototypes.Any{
+			// TODO get type url dynamically(basically remove `+ "Response"`) after the following issue has solved.
+			// https://github.com/cosmos/cosmos-sdk/issues/10496
```

Comment:
```suggestion
			// TODO get type url dynamically(basically remove `+ "Response"`) after the following issue has solved.
			// https://github.com/ignite/cli/issues/2098
			// https://github.com/cosmos/cosmos-sdk/issues/10496
```
I know it's not related to this PR but I think we should put in comment the associated tracked issue for TODOs so that we can notify the issue if something changed on the related code. It doesn't seem to be particularly the case here